rfcs/0005-moxy-carousel.md

```diff
@@ -0,0 +1,195 @@
+        {slidesData.map((data) => (
+            <Slide key={ data.title } { ...data } />
+        ))}
+    </Carousel>
+);
+
+export default Home;
+```
+
+- `Home.module.css`
+
+```css
+@import "../../shared/styles/variables";
+
+.home {
+    width: var(--home-width);
+    height: var(--home-height);
+    background-color: var(--color-white);
+}
+
+.containerClass {
+    margin-top: 200px;
+
+    & .itemClass {
+        display: flex;
+        justify-content: center;
+
+        opacity: 0.3;
+
+        & .slide {
+            width: 100%;
+            height: 200px;
+
+            display: flex;
+            justify-content: center;
+            align-items: center;
+
+            color: whitesmoke;
+            font-weight: 700;
+            text-transform: uppercase;
+        }
+
+        & .firstSlide {
+            background-color: hotpink;
+        }
+
+        & .secondSlide {
+            background-color: purple;
+        }
+
+        & .thirdSlide {
+            background-color: darkcyan;
+        }
+
+        & .forthSlide {
+            background-color: darksalmon;
+        }
+    }
+
+    & .activeItemClass {
+        opacity: 1;
+    }
+}
+```
+
+# Motivation
+
+Everytime we need to implement a carousel/slider we are dependant on external libs, usually these libs meet the basic requirements, but eventually they will become problematic for more extended use cases.
+
+It would be of great value to have our own implemented carousel/slider to prevent extended overhead time when implementing the wrapper for the external module.
+
+# Detailed design
+
+The original `react-multi-carousel` was lacking a way to style the active item(s) without overriding the package's CSS, so I added a new prop called `activeItemClass` that accepts a `class`.
+
+More changes and/or additions will come after discussion about our needs.
+
+# Drawbacks
+
+- Depending on how many new features we will add, this will drastically diverge from upstream and thus it will require more effort to maintain;
+- The original codebase is written in TypeScript (although I don't think this is technically a drawback, it should be noted);
+- Not compatible with React Native;
+- There are 13 contributors, but it seems that the author has been doing almost all of the work.
+
+# Alternatives
+
+- [nuka-carousel](https://github.com/FormidableLabs/nuka-carousel)
+- [react-slick](https://github.com/akiran/react-slick)
+- [material-auto-rotating-carousel](https://github.com/TeamWertarbyte/material-auto-rotating-carousel)
+
+# Adoption strategy
+
+If we implement this proposal we will need to have in mind that it will require some time to maintain it.
+
+# Unresolved questions
+
+I'm not sure about [removing the original `react-multi-carousel-item--active`](https://github.com/moxystudio/react-carousel/commit/4f7b87429775f18e455779de38d3b8d64647a688) as the default `class` for an active item. Maybe I should keep it and simply append the `class` given through the `activeItemClass`:
```

Comment:
Doing that is better. More changes that the PR will be accepted by the repo's owner.

Reply:
Done! https://github.com/moxystudio/react-carousel/pull/3